src/test/scala/com/cognite/spark/datasource/TimeSeriesRelationTest.scala

```diff
@@ -243,8 +275,46 @@ class TimeSeriesRelationTest extends FlatSpec with Matchers with SparkTest {
   }
 
   it should "support upsert in savemode" taggedAs WriteTest in {
+    val insertDescription = "spark-insert-test"
     val upsertDescription = "spark-upsert-test"
-    val saveModeUnit = "spark-savemode-test"
+
+    // Clean up any old test data
+    cleanUpTimeSeriesTestDataByUnit(testDataUnit)
```

Comment:
Just curious, why did this test change so much? We had just forgotten a cleanup in here or something?

Reply:
Several of the tests in `TimeSeriesRelationTest` relied on data from previous tests still existing, and being in the state left from the previous test. This was annoying when wishing to rerung a single test, so I decided to fix it for all of them.